### PR TITLE
Callback function after rendering is completed

### PR DESCRIPTION
**Pull request by jeevasusej**
Callback function after rendering is completed

### CODE REVIEW

**Comment by carlhussey** [comment]
@jeevasusej  - How is this used? Is this callback made once the whole chart has been rendered to the DOM ?

**Comment by plinioasf** [comment]
@carlhussey you need to pass option 'renderingCompleted' in the orgchart call and pass a function that will receive the orgchart object

File: dist/js/jquery.orgchart.js [hunk 1]
```diff
@@ -288,6 +288,8 @@
       });
     }
 
+	if (opts.renderingCompleted && typeof opts.renderingCompleted === "function")
+      opts.renderingCompleted($chartContainer);
```
Comment (plinioasf) [inline dist/js/jquery.orgchart.js]: When ajax is used $chartContainer will be probably empty..

**Comment by dabeng** [comment]
Hi @jeevasusej @carlhussey @plinioasf , thanks a lot for your enthusiastic discussions, you can make use of event init and callback initCompleted since orgchart v1.4.0 :blush:

**Comment by dabeng** [comment]
I append the following code snippets to trigger initCompleted callback and emit init event:

```js
    var mo = new MutationObserver(function (mutations) {
      mo.disconnect();
      initTime:
      for (var i = 0; i < mutations.length; i++) {
        for (var j = 0; j < mutations[i].addedNodes.length; j++) {
          if (mutations[i].addedNodes[j].classList.contains('orgchart')) {
            if (opts.initCompleted && typeof opts.initCompleted === 'function') {
              opts.initCompleted($chart);
              $chartContainer.triggerHandler({ 'type': 'init.orgchart', 'chart': $chart });
              break initTime;
            }
          }
        }
      }
    });
    mo.observe($chartContainer[0], { childList: true });
```

**Comment by dabeng** [comment]
Hope you guys like the above solution.

**Comment by dabeng** [comment]
Everyone interested in this callback can play around this [Demo](https://codepen.io/dabeng/pen/vZjpaw)

**Comment by Seema5** [comment]
Hello Everyone
I have one issue while using jquery orgchart plugin.
i want to add two children at every node if there is no children. These two children should aligned at left or right as undefined node. Or if node have one child then it also have another children at left or right as undefined node
             Thankyou

**Pull request closed without merging**